Codes/Third Module/module_attributes.exs

```diff
@@ -0,0 +1,16 @@
+defmodule My_module do
```

Comment:
Instead of **My_module** you should use something like **MyModule** to follow the UpperCamelCase convention